FWCore/PythonFramework/src/PythonEventProcessor.cc

```diff
@@ -66,18 +67,14 @@ PythonEventProcessor::PythonEventProcessor(PyBind11ProcessDesc const& iDesc)
 
 PythonEventProcessor::~PythonEventProcessor() {
   auto gil = PyEval_SaveThread();
-  try {
-    processor_.endJob();
-  } catch (...) {
+  CMS_SA_ALLOW try { processor_.endJob(); } catch (...) {
   }
   PyEval_RestoreThread(gil);
 }
 
 void PythonEventProcessor::run() {
   auto gil = PyEval_SaveThread();
-  try {
-    (void)processor_.runToCompletion();
-  } catch (...) {
+  CMS_SA_ALLOW try { (void)processor_.runToCompletion(); } catch (...) {
```

Comment:
Python can not convert C++ exception.

Reply:
@gartung After some more thought we came to the conclusion that it might be possible to remove this try-catch, but let's leave that to future work. For this PR, please remove the `CMS_SA_ALLOW` here, and I'll create an issue to follow up.